sdk/communication/Azure.Communication.MediaComposition/src/Generated/Models/CommunicationIdentifierModel.cs

```diff
@@ -16,17 +16,29 @@ public CommunicationIdentifierModel()
         }
 
         /// <summary> Initializes a new instance of CommunicationIdentifierModel. </summary>
-        /// <param name="communicationUser"> A user that got created with an Azure Communication Services resource. </param>
-        /// <param name="microsoftTeamsUser"> A Microsoft Teams user. </param>
-        internal CommunicationIdentifierModel(CommunicationUserIdentifierModel communicationUser, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser)
+        /// <param name="kind"> The identifier kind. Only required in responses. </param>
+        /// <param name="rawId"> Raw Id of the identifier. Optional in requests, required in responses. </param>
+        /// <param name="communicationUser"> The communication user. </param>
+        /// <param name="phoneNumber"> The phone number. </param>
+        /// <param name="microsoftTeamsUser"> The Microsoft Teams user. </param>
+        internal CommunicationIdentifierModel(CommunicationIdentifierModelKind? kind, string rawId, CommunicationUserIdentifierModel communicationUser, PhoneNumberIdentifierModel phoneNumber, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser)
         {
+            Kind = kind;
+            RawId = rawId;
             CommunicationUser = communicationUser;
+            PhoneNumber = phoneNumber;
```

Comment:
Did we decide to just use the CommunicationIdentiferModel which includes phone number?